### PR TITLE
LW-7770 Prevent from adding more than 5 pools in Browse Pools

### PR DESCRIPTION
**Pull request by mrcnk**
# Checklist

- [x] [JIRA](https://input-output.atlassian.net/browse/LW-7770)
- [ ] Proper tests implemented
- [x] Screenshots added.

---

## Proposed solution

- Adds limit set to 5 as max pools to add to draft in hover stake pool buttons.

## Screenshots

image

---
# Allure report
`allure-report-publisher` generated test report!

**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/linux/chrome/1742/6000657468/index.html) for [4f7cc37e](https://github.com/input-output-hk/lace/pull/396/commits/4f7cc37ef74e97d2ee0fa653d91f4b98099e071b)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 34     | 1      | 0       | 0     | 35    | ❌     |

### CODE REVIEW

File: packages/staking/src/features/browse-pools/stake-pools-table/StakePoolItemBrowser/StakePoolItemBrowser.tsx [hunk 1]
```diff
@@ -9,6 +9,8 @@ import { useTranslation } from 'react-i18next';
 import { useDelegationPortfolioStore } from '../../../store';
 import styles from './StakePoolItemBrowser.module.scss';
 
+const MAX_ITEMS_IN_DRAFT = 5;
```
Comment (szymonmaslowski) [inline packages/staking/src/features/browse-pools/stake-pools-table/StakePoolItemBrowser/StakePoolItemBrowser.tsx:12]: This value is already defined in the `delegationPortfolio` store. Please import it instead

**Review by szymonmaslowski — CHANGES_REQUESTED** [review]

File: packages/staking/src/features/store/delegationPortfolio.ts [hunk 2]
```diff
@@ -13,12 +13,16 @@ export const MAX_POOLS_COUNT = 5;
 export const useDelegationPortfolioStore = create(
   immer<DelegationPortfolioStore>((set, get) => ({
     ...defaultState,
+    ableToAddMorePools: () => {
+      const { draftPortfolio } = get();
+      return draftPortfolio.length < MAX_POOLS_COUNT;
+    },
```
Comment (przemyslaw-wlodek) [inline packages/staking/src/features/store/delegationPortfolio.ts:19]: Please make it a selector, not a query. 🙏🏻

**Review by przemyslaw-wlodek — CHANGES_REQUESTED** [review]

File: packages/staking/src/features/store/delegationPortfolio.ts (hunk 2 above)
Comment (przemyslaw-wlodek) [reply packages/staking/src/features/store/delegationPortfolio.ts:19]: As discussed offline. There is another query that should become a selector. Let's tackle that in another PR 👍🏻

**Review by przemyslaw-wlodek — APPROVED** [review]

**Comment by ManusMcCole** [comment]
Hi @mrcnk. Looks good almost ready to approve. One last thing though could we make the tooltip closer to the Figma design. There is a lot of usused space in the tooltip towards the right hand side.Not sure if there was a conversation about how it hovers over the button.
Screenshot 2023-08-25 at 11 46 07
Figma = 
Screenshot 2023-08-25 at 11 42 40

**Review by ManusMcCole — APPROVED** [review]

**Pull request merged**